packages/core/utils/src/modules-sdk/migration-scripts/migration-down.ts

```diff
@@ -34,34 +28,30 @@ export function buildRevertMigrationScript({
   > = {}) {
     logger ??= console as unknown as Logger
 
-    const dbData = loadDatabaseConfig(moduleName, options)!
-    const entities = Object.values(models).map((model) => {
-      if (DmlEntity.isDmlEntity(model)) {
-        return toMikroORMEntity(model)
-      }
+    console.log(new Array(TERMINAL_SIZE).join("_"))
+    console.log("")
```

Comment:
Q: any reason to not use the logger? is it to prevent it from appearing in the json log?

Reply:
The logger has a prefix of `info` by default and therefore it will show up in the empty line and same for the `-`